static/scss/detail/learning-outcomes.scss

```diff
@@ -0,0 +1,56 @@
+// sass-lint:disable mixins-before-declarations
+
+.learning-outcomes-block {
+  background: white url('#{$static-path}/images/bg-access.png') repeat-x 0 0;
+  padding: 50px 0;
+
+  .heading-container {
+    margin: 0 auto;
+    max-width: 1000px;
+    text-align: center;
+    font-size: 28px;
+    line-height: 38px;
+    font-weight: 500;
```

Comment:
Please get rid of `font-size`, `line-height`, and `font-weight` from this definition. The heading and subheading are the only 2 children of the `heading-container` element and they do not seem to share common styles for those properties